### PR TITLE
Fix/improve linter

### PR DESCRIPTION
**Pull request by Harian-Elyoth**

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,2 +1 @@
-npm run lint
-git add .
+npx --no-install lint-staged
```
Comment (hcanon42) [inline .husky/pre-commit:1]: cool :)

**Review by hcanon42 — COMMENTED** [review]
Code seems good to me!
One thing I am not sure about is the number of commits though, seems like this is one "feature" and breaking it into multiple commits complexifies the git tree and will make it harder in 6 months if we need to reset some things.
I understand for APIs like one commit for the feature and one for the tests why not but here I don't really get the point...

File: .husky/pre-commit (hunk 1 above)
Comment (kimdahee0815) [reply .husky/pre-commit:1]: it works pretty well :-)

**Review by kimdahee0815 — APPROVED** [review]
code works well 👍

**Pull request merged**